.github/workflows/codechecker.yml

```diff
@@ -0,0 +1,95 @@
+name: Run CodeChecker static analyzer on XAPI's C stubs
+
```

Comment:
```suggestion
permissions: {}
```

Reply:
There is a declaration of permissions on the job itself, but it can't hurt to add a global one too.